packages/backend/src/open_payments/grant/model.ts

```diff
@@ -36,8 +38,7 @@ export class Grant extends BaseModel {
     return !!this.expiresAt && this.expiresAt <= new Date()
   }
 
-  $afterFind(queryContext: QueryContext): void {
-    super.$afterFind(queryContext)
-    delete this['authServer']
+  public get managementUrl(): string {
+    return `${this.authServer?.url}/token/${this.managementId}`
```

Comment:
```suggestion
    return `${this.authServer.url}/token/${this.managementId}`
```
since we have `public authServer!: AuthServer` required above

If `authServer` is actually optional, then I think `managementUrl` should throw or return `undefined` if it's missing

Reply:
In my opinion, `authServer` should be required.